aztec-up/bin/.aztec-run

```diff
@@ -7,7 +7,12 @@ set -euo pipefail
 IMAGE=${1:-}
 shift
 
+DEFAULT_PORT=8080
 VERSION=${VERSION:-"latest"}
+AZTEC_PORT=${AZTEC_PORT:-$DEFAULT_PORT}
+
+# preserve arguments to pass to docker run
+declare -a preserved_args
```

Comment:
Does this work in the old bash version that ships with macos?

Reply:
I don't see this variable being used.

Reply:
Good catch on the usage, think I need to use as input on the `docker run` command.
Will check about compatibility